<launch>
<node name="px4_velocity_rc_control_wrapper" pkg="px4_control_wrapper" type="px4_velocity_rc_control_wrapper" output="screen">
      <param name="px4_ref_vel_msg_pub_topic" value="/mavros/setpoint_velocity/cmd_vel_unstamped"/>        <!--The Target Velocity Message to PX4-->
      <param name="tgt_vel_msg_sub_topic" value="/other_controller/target_velocity"/>                      <!--The Target Velocity from Other Controller-->
      <param name="px4_rc_msg_sub_topic" value="/mavros/rc/in"/>                                           <!--The RC Information from PX4-->
      <param name="run_frequen_hz" value="20.0"/>                                                          <!--The Frequen of Target Velocity Message To PX4-->
      <param name="rc_start_ch" value="5"/>                                                                <!--The RC Channel of Start Vehicle-->
      <param name="rc_land_ch" value="6"/>                                                                 <!--The RC Channel of Land Vehicle-->
      <param name="is_check_data_valid" value="false"/>
      <param name="data_valid_sub_topic" value="/uav0/data/valid"/>
      <param name="is_local_pos" value="false"/>
      <param name="uav_pos_sub_topic" value="/mavros/local_position/pose"/>
      <param name="uav_local_pos_pub_topic" value="/uav/local_position/local_pose"/>
</node>
</launch>
